Cliqz/Search/LumenSearchEnginePicker.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  LumenSearchEnginePicker.swift
+//  Client
+//
+//  Created by Pavel Kirakosyan on 15.07.19.
+//  Copyright © 2019 Cliqz. All rights reserved.
+//
+
+import UIKit
+import Shared
+
+class LumenSearchEnginePicker: SearchEnginePicker {
+
+    var profile: Profile?
+    var searchEnginesUpdated: (() -> Void)?
+
+    // MARK: - Table view data source
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = NSLocalizedString("Default Search Engine", comment: "Title for default search engine picker.")
+    }
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // 1 engines, 2 create new
+        return 2
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if section == 0 {
+            return super.tableView(tableView, numberOfRowsInSection: section)
+        }
+
+        return 1
+    }
+
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if indexPath.section == 0 {
+            return super.tableView(tableView, cellForRowAt: indexPath)
+        }
+
+        let cell = ThemedTableViewCell()
```

Comment:
You should [register](https://developer.apple.com/documentation/uikit/uitableview/1614888-register) the cell types you need at table load and then [dequeue](https://developer.apple.com/documentation/uikit/uitableview/1614878-dequeuereusablecell) them when you need it. It's not super important here, but let's do it right